src/pb_error.h

```diff
@@ -32,21 +32,40 @@
 #include "pb_utils.h"
 
 namespace triton { namespace backend { namespace python {
+
+struct PbErrorShm {
+  TRITONSERVER_Error_Code code;
+  bi::managed_external_buffer::handle_t message_shm_handle;
+};
+
 class PbError {
  public:
-  PbError(const std::string& message) : message_(message) {}
+  PbError(const std::string& message);
+  PbError(TRITONSERVER_Error_Code code, const std::string& message);
```

Comment:
I think we can merge these two constructors into one by making code the second argument and providing a default value of internal

Reply:
Sure. Updated: [Unify PbError constructors](https://github.com/triton-inference-server/python_backend/pull/292/commits/6c80fab403e56599c379f15dbe32e643c487fb20)